a11y/prefers-reduced-motion/demos/disable-animation/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+  <title>Отключение анимации — Как управлять анимацией на сайте и сделать её доступной — Дока</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto&display=swap">
+</head>
+<style>
+  body {
+    background-color: #282a2e;
+    overflow: hidden;
+  }
```

Comment:
Очень здорово! 😊 Мне кажется, нян кэт просто идеален для этой статьи :)

Давайте марджин сбросим?

```suggestion
  body {
    margin: 0;  
    background-color: #282a2e;
    overflow: hidden;
  }
```